WordPressAuthenticator/Extensions/Collection+AuthHelpers.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+
+extension Collection {
```

Comment:
This one is already present in WordPRess-iOS 'CollectionType+Helpers.swift'; even if that cannot be directly used here, I am wondering if it would be the case of putting this utility in a shared dependency (`WordPressShared` maybe?) instead of duplicating the code. What do you think?